mkdocs.yml

```
@@ -15,7 +15,9 @@ nav:
       - Features:
           - core/tracer.md
           - core/logger.md
-          - core/metrics.md
+          - Metrics:
+              - core/metrics.md
+              - core/metrics_datadog.md
```

```suggestion
          - Metrics:
              - core/metrics/index.md
              - Amazon CloudWatch EMF: core/metrics.md
              - Datadog: core/metrics_datadog.md
```

This allows customers to click `Metrics` in case of hitting the arrow (no UX degradation), and still have the ability to select `Amazon CloudWatch EMF` without any content deduplication from our side by using an append.

and the `core/metrics/index.md` content being:

```markdown
---
title: Metrics
description: Core utility
---

--8<-- "docs/core/metrics.md"

```